.peoplemap

```diff
@@ -49,6 +49,7 @@ Radoslaw Zarzynski <rzarzyns@redhat.com> <rzarzynski@mirantis.com>
 Robin H. Johnson <rjohnson@digitalocean.com> <robin.johnson@dreamhost.com>
 Ross Turk <rturk@redhat.com> Ross Turk <ross.turk@inktank.com>
 Sage Weil <sweil@redhat.com> Sage Weil <sage@inktank.com>
+Sebastian Wagner <sewagner@redhat.com> Sebastian Wagner <swagner@suse.com>
```

Comment:
not sure about this one. Kefu?

Reply:
i think this is correct. as this line reads: 

> `Sebastian Wagner <sewagner@redhat.com>` and `Sebastian Wagner <swagner@suse.com>` is actually the same person

see also https://tracker.ceph.com/projects/ceph/wiki/Ceph_contributors_list_maintenance_guide#Fixing-the-contributor-list and https://github.com/ceph/ceph/blob/9c00a343e01f7bd1dfae300f991b32ccc69fcca3/.peoplemap#L15-L17